spec/src/main/asciidoc/appendix.adoc

```diff
@@ -127,6 +127,40 @@ public class MetricRegistryFactory {
 
 === Migration hints
 
+[[migration-hint-to-50]]
+==== To version 5.0
+
+===== SimpleTimer / @SimplyTimed
+
+The `SimpleTimer` class and `@SimplyTimed` annotation have been removed.  This change was made to make it possible to implement the spec using commonly used metrics libraries that lack a similar metric type.
+
+Use `Timer` class or `@Timer` annotation instead.  Alternatively, you can create your own `Gauge` to track the total time and your own `Counter` to track the total number of hits of something you want to time.
```

Comment:
isn't the annotation still `@Timed` (ending with 'd' not 'r')?

Reply:
oops. yes!